styles.css

```diff
@@ -59,25 +59,60 @@ body {
     margin-bottom: 5px;
     border: 1px solid #ddd;
     border-radius: 5px;
+   
   }
 
-  
+/*   
 button {
   padding: 5px 10px;
   border: none;
   border-radius: 4px;
   cursor: pointer;
-}
+} */
```

Comment:
What is the need to comment on this? It's not good to comment out the code without any specific reason. If you feel something is irrelevant, you can delete it without harming any default functionality.